app.rb

```diff
@@ -1,9 +1,33 @@
 require 'sinatra/base'
+require 'sinatra/reloader'
+require './lib/chitter'
 
 class Chitter < Sinatra::Base
+  configure :development do
+    register Sinatra::Reloader
+  end
+
   get '/test' do
     'Test page'
   end
 
+  get '/messages' do
+    @messages = ChitterMessage.all
+    erb :index
+  end
+
+  get '/new' do
```

Comment:
This route name needs to be more explicit as it is not clear what new is referring to. Instead, **get /messages/new** would be much clearer.